packages/@sanity/base/src/_exports/hooks.ts

```diff
@@ -1,12 +1,16 @@
 export {useDocumentPresence, useGlobalPresence} from '../datastores/presence/hooks'
 export {useUser, useCurrentUser} from '../datastores/user/hooks'
 export {useDocumentType} from '../datastores/document/hooks'
+/* eslint-disable camelcase */
 export {
-  // eslint-disable-next-line camelcase
-  unstable_useCheckDocumentPermission,
-  // eslint-disable-next-line camelcase
-  unstable_useCanCreateAnyOf,
-} from '../datastores/grants/hooks'
+  unstable_getDocumentPairPermissions,
+  unstable_getDocumentValuePermissions,
+  unstable_getTemplatePermissions,
+  unstable_useDocumentPairPermissions,
+  unstable_useDocumentValuePermissions,
+  unstable_useTemplatePermissions,
```

Comment:
could we also export the `TemplatePermissionsResult` type from either here or the `_internal` exports?